data/is_ocaml_yet/gui.md

```diff
@@ -0,0 +1,74 @@
+---
+id: gui
+question: Is OCaml GUI Yet?
+answer: Not quite yet, but we're getting there!
+categories:
+  - name: Frameworks
+    status: 🟡
+    description: |
+      These are libraries written entirely in OCaml. It has packages for syndicating content and working with RSS feeds.
+    packages:
+      - name: bogue
+      - name: lablgtk
+      - name: lablqml
+      - name: tsdl
+      - name: sfml
+  - name: 3d graphics
+    status: 🟡
+    description: |
+      These are libraries that are for manipulating 3D graphics. They offer a wide range of functionalities, including geometric transformations, rendering, shading, and handling user input.
+    packages:
+      - name: tgls
+      - name: glMLite
+      - name: irrlicht
+  - name: 2d graphics
+    status: 🟡
+    description: |
+      These libraries provides the necessary functionalities to create and manipulate 2D graphics.
+    packages:
+      - name: vg
+      - name: gg
+      - name: async_graphics
+      - name: cairo2
+  - name: Web (Js bindings)
+    status: 🟡
+    description: |
+      These libraries are used to create bindings between OCaml and JavaScript, allowing OCaml code to interact with JavaScript libraries. They help in creating web applications using OCaml by providing a bridge between OCaml and JavaScript.
+    packages:
+      - name: chartjs
+  - name: image processing
+    status: 🟢
+    description: |
+      These libraries provide functionalities to manipulate and process images. They offer a range of functionalities for image processing tasks, like basic image manipulation to more complex operations. They make OCaml a powerful language for image processing.
+    packages:
+      - name: tsdl-image
+      - name: bimage
+      - name: ocamlsdl2-image
+  - name: Plotting
+    status: 🟢
+    description: |
+      These libraries are used to create visual representations of data. They provide functionalities to generate various types of plots, charts, and diagrams.
+    packages:
+      - name: plot
+      - name: owl-plot
+      - name: oplot
+      - name: plotty
```

Comment:
`owl-plot` and `plotty` don't seem to be on opam-repository, so they would need to be linked as github links (see `web.md` document).

Reply:
Agreed, @IdaraNabuk the syntax is this one: https://github.com/ocaml/ocaml.org/blob/main/data/is_ocaml_yet/web.md?plain=1#L28-L31

Reply:
I just checked, I don't know why I've listed `plotty` either, I couldn't find it anywhere, let's just remove it from the list!
And `owl-plot` should be `owl-plplot`, but by checking the owl repository I'm not sure this package should be listed either (looks like something internal that got published on opam by mistake). So let's remove them both. If people want to add more libraries to the list in the future they can open a PR.